src/System.Windows.Forms.Design/src/System/Windows/Forms/Design/TabOrder.cs

```diff
@@ -294,18 +265,19 @@ private void DrawTabs(IList tabs, Graphics gr, bool fRegion)
                 rc.Width = sz.Width + 2;
                 rc.Height = sz.Height + 2;
 
+                Debug.Assert(tabGlyphs is not null, "tabGlyps should not be null here.");
                 tabGlyphs[iCtl++] = rc;
 
                 Brush brush;
                 Pen pen;
                 Color textColor;
                 if (fRegion)
                 {
-                    region.Union(rc);
+                    region?.Union(rc);
                 }
                 else
                 {
-                    if (tabComplete.IndexOf(ctl) != -1)
+                    if (tabComplete is not null && !tabComplete.Contains(ctl))
```

Comment:
I think you reversed the condition here
```suggestion
                    if (tabComplete is not null && tabComplete.Contains(ctl))
```

Reply:
Good Catch!